### ISSUE TITLE
After I install the extension, please let me know what I need to do

### ISSUE THREAD

**Issue by redox** [algolia/github-awesome-autocomplete#12]

**Comment by tlvenn** [comment]
Seems like a good idea, I am struggling to understand where I am supposed to `connect with github` ? There is no such action from the plugin interface.

**Comment by redox** [comment]
> Seems like a good idea, I am struggling to understand where I am supposed to connect with github ? There is no such action from the plugin interface.

yes, definitely something we need to improve. Thank you for your inputs @tlvenn !
